apps/web/src/app/[locale]/page.tsx

```diff
@@ -40,7 +40,9 @@ export default async function Home({
         images={landingPageData.heroImages
           .map(({ image }) => (typeof image === "string" ? null : image?.url))
           .filter((url): url is string => Boolean(url))}
-        text={landingPageData.heroText}
+        texts={landingPageData.heroText
```

Comment:
Since we don't have proper migrations this will shortly break our production landing page before we update the collection in the CMS :sweat_smile: 

It's ok this time, but when it's in real production we would probably need to add a runtime check that the field is an array, or similar if we're making breaking changes to collections